NEWS.md

```diff
@@ -7,7 +7,7 @@
 ## Changes to defaults
 
 * `brace_linter()` allows opening curly braces on a new line when there is 
-  a comment on the preceding line (#1451, @IndrajeetPatil).
+  a comment ending the preceding line (#1433 and #1434, @IndrajeetPatil).
```

Comment:
I think all of these are wrong for PRs I've contributed because I thought we were writing the PR numbers that fixed the issue, and not the issue numbers. 🤦‍♂️

I will make a separate PR to fix all of them. Sorry about that.

Reply:
I don't think it matters enough to make a cleanup PR -- I just think it's more direct for NEWS reader to land on the bug report page than the PR page (which will link tot he bug page).

the implicit assumption is the reader cares about the nature of the bug (to see if they're affected) more than the code to fix it.